packages/@okta/vuepress-site/docs/reference/api/factors/index.md

```diff
@@ -1545,6 +1546,99 @@ curl -v -X POST \
 }
 ```
 
+#### Enroll WebAuthn Factor
+
+Enrolls a user with a WebAuthn factor. The enrollment process starts with getting the WebAuthn credential creation options, which are used to help select an appropriate authenticator using the WebAuthn API.
+This authenticator then generates an enrollment attestation which may be used to register the authenticator for the user.
+
+##### Enroll WebAuthn Response Parameters
+
+In the [embedded resources](#embedded-resources) object, the `response._embedded.activation` object contains properties used to guide the client in creating a new WebAuthn credential for use with Okta.
+
+For more information about these credential creation options, see the [WebAuthn spec for PublicKeyCredentialCreationOptions](https://www.w3.org/TR/webauthn/#dictionary-makecredentialoptions)
```

Comment:
add a period

For more information about these credential creation options, see the [WebAuthn spec for PublicKeyCredentialCreationOptions](https://www.w3.org/TR/webauthn/#dictionary-makecredentialoptions).